pkg/resourceinterpreter/customized/webhook/configmanager/accessor_test.go

```diff
@@ -0,0 +1,217 @@
+/*
+Copyright 2024 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configmanager
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	webhookutil "k8s.io/apiserver/pkg/util/webhook"
+	"k8s.io/utils/pointer"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+)
+
+func TestNewResourceExploringAccessor(t *testing.T) {
```

Comment:
This test case has been overwritten by the next test case and can be deleted.

Reply:
Hi @XiShanYongYe-Chang , Should i add these two test cases to the getters test or remove it completely?

Reply:
> Should i add these two test cases to the getters test

It would be great to do this.